nucliadb/src/nucliadb/common/cluster/rebalance.py

```diff
@@ -68,32 +68,32 @@ def __init__(self, context: ApplicationContext, kbid: str):
         self.context = context
         self.kbid = kbid
         self.kb_shards: Optional[writer_pb2.Shards] = None
-        self.index: dict[str, int] = {}
 
-    async def get_rebalance_shards(self) -> list[RebalanceShard]:
+    async def get_rebalance_shards(self, dirty: bool = False) -> list[RebalanceShard]:
```

Comment:
Maybe `dirty` could be called something like `estimate`? I think it might be clearer